packages/govuk-frontend/src/govuk/components/character-count/character-count.test.js

```diff
@@ -648,7 +670,7 @@ describe('Character count', () => {
           // Override maxlength to 10
           maxlength: 10
         },
-        initialiser ($module) {
+        beforeInitialisation ($module) {
```

Comment:
As it's no longer with us, as a sign of respect for `.init()` shall we rename this? 😆

Suppose it matches `initAll()` a bit better too

```suggestion
        beforeInit ($module) {
```